api/search_test.go

```diff
@@ -380,7 +423,8 @@ func TestSearchHandlerFunc(t *testing.T) {
 				"&dimensions=dim1,dim2"+
 				"&population_types=pop1,pop2"+
 				"&fromDate=2020-10-10"+
-				"&toDate=2023-10-10",
+				"&toDate=2023-10-10"+
+				"&dataset_id=QNA,QDA",
```

Comment:
Given the other ones that accept multiples are pluralised, e.g. dimensions, population_types, do you think this should be `dataset_ids`